### ISSUE TITLE
Creating a second new game spawns planets with no resources

### ISSUE THREAD

**Issue by WA-Wink** [Touhma/DSP_Galactic_Scale#233]
I was playing around with the latest mod updates and created a new game. I started the game and decided I did not like the planet. I quit to menu and created another new game. The universe creator shows the systems with only liquid resources, no iron/copper/coal/stone/etc.

Repro steps:
1) Create a new game using GS2
2) Start the game
3) Quit to menu
4) Create a new game
5) View the system (or planet) resources from the new game screen

Actual:
No solid resources, only liquid

Expected:
Standard resources on planets

**Comment by innominata** [comment]
Thanks for the detail, will look into it!

**Comment by WA-Wink** [comment]
I just did another quick test, while the universe creation screen does not show resources, the resources are there when you start the game so it is a display issue on the creation screen.

**Comment by innominata** [comment]
Yeah the game has to generate all the planet terrain before it can place ore. It might be as simple as it hasn't generated the ore yet. Base game doesn't have the problem as it generates on the fly and the player takes time to move between systems / get the required tech to see at a distance.

**Comment by AndrewLuebke** [comment]
Is this still occurring on 2.78.5 (current)? The repro recipe (quit to menu, create a second new game, resources missing in the universe view) is clear enough to chase if it still happens — please confirm on the current version and we'll take it from there.

**Comment by WA-Wink** [comment]
Still seeing it. Not a big deal, just a minor issue unless you are trying
to start on a planet/system with certain resources. I just went through and
disabled all mods except GS and it still is occurring.

[image: image.png]

**Comment by AndrewLuebke** [comment]
Thanks for the clean isolation — that confirms it's ours. The repro pattern (first game fine, second game's universe view missing ore veins but keeping liquids) points at generator state that survives quit-to-menu when it shouldn't; we're digging into it now and this issue is on the active fix list.

**Comment by AndrewLuebke** [comment]
Fixed — merged as #289, will be in the next release. Root cause: quitting to the menu sets an internal "quit in progress" flag that GS2's galaxy-preview generation checked but nothing on the menu path ever cleared, so every galaxy preview after the first quit silently skipped placing ore veins (oceans and gas come from a different path, which is why liquids survived). Verified with your exact recipe: reproduced on stock 2.78.5, gone with the fix. Thanks for the clean isolation — it made this quick to pin down.

**Comment by AndrewLuebke** [comment]
@WA-Wink this fix is now released — **GalacticScale 2.78.6** is live on Thunderstore. Could you update and confirm the second/third new game in a session now generates veins normally? If you still see empty planets on 2.78.6, please reopen with a note on which mods were enabled and we'll dig back in. Thanks for the clear report and the mod-isolation testing — it's what pinned this down.
